### PR TITLE
Readme illustration

### LINKED ISSUE #151: add illustration to README

### PR DESCRIPTION
**Pull request by McHaillet**
Added a cover image to the readme :) 
Closes #151

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 ![test-badge](https://github.com/SBC-Utrecht/pytom-match-pick/actions/workflows/unit-tests.yml/badge.svg?branch=main)
-# GPU template matching for cryo-ET
+# pytom-match-pick: GPU template matching for cryo-ET
 
 GPU template matching, originally developed in [PyTom](https://github.com/SBC-Utrecht/PyTom), as a standalone python package that can be run from the command line.
 
+![cover_image](images/tomo200528_100_illustration.png)
```
Comment (sroet) [inline README.md:6]: This does not look OK in the rich diff, but does look ok when I checkout the branch on github

**Review by sroet — APPROVED** [review]
LGTM! Feel free to merge (added one comment because I was confused by it)

File: README.md (hunk 1 above)
Comment (McHaillet) [reply README.md:6]: for me it looks fine in the rich diff, what looks off for you?

Comment (sroet) [reply README.md:6]: It looked like a regular link (like the Pytom one on the line above it)

**Pull request merged**